tests/tasks/setup_test.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Include general tests variables
+  include_vars: main.yml
+
```

Comment:
prefer only 1 empty line between tasks instead of 2

Reply:
is there a way to enforce this in yamllint?

Reply:
> is there a way to enforce this in yamllint?

not sure - will investigate

Reply:
hmm - I don't think this is a yaml thing, I think it is an ansible thing - https://docs.ansible.com/ansible-lint/ doesn't seem to have a check for that @pcahyna @tyll do you know if there is a linter for this?  OTOH, I don't recall if 'one blank line between tasks' is in any of the best practices, but it is a common convention.

Reply:
I hear about this for the first time. Would be great to have a linter or actually formatter for this.